DEPRECATION.md

```diff
@@ -0,0 +1,30 @@
+# Deprecation Policy
+
+Any undocumented API or function is subject to change at any time.  Only documented APIs are subject to the following 
+policy.
```

Comment:
This goes against what we've defined previously in the dev-policy, with regards to "implicit" and "explicit" public APIs, IIRC.